### PR TITLE
Fixed issue #16782: Survey groups : theme options: same theme name appear multiple time in list

### PR DESCRIPTION
**Pull request by gabrieljenik**
Fixing issue and deleting wrongly generated records: duplicate template configurations. Keeping the first one.

### CODE REVIEW

**Comment by Shnoulle** [comment]
Original issue was fixed ? No way to get multiple one ?

**Comment by gabrieljenik** [comment]
> Original issue was fixed ? No way to get multiple one ?

We think issue is solved and duplicates shouldn't be created (in relation to this issue/situation).
Why...? Have found a crack?

File: application/helpers/update/updatedb_helper.php [hunk 1]
```diff
@@ -3783,12 +3783,29 @@ function ($v) {
...
+            $deleteQuery = "DELETE FROM {{template_configuration}}
+                WHERE id NOT IN (
+                    SELECT id FROM (
+                        SELECT MIN(id) as id
+                            FROM {{template_configuration}} t 
+                            GROUP BY t.template_name, t.sid, t.gsid, t.uid
+                    ) x
+                )";
+            $oDB->createCommand($deleteQuery)->execute();
+            $oDB->createCommand()->update('{{settings_global}}', array('stg_value' => 442), "stg_name='DBVersion'");
```
Comment (c-schmitz) [inline application/helpers/update/updatedb_helper.php:3806]: Wrong DBVersion here

**Review by c-schmitz — CHANGES_REQUESTED** [review]

File: application/helpers/update/updatedb_helper.php (hunk 1 above)
Comment (gabrieljenik) [reply application/helpers/update/updatedb_helper.php]: Fixed. Thanks!

**Pull request merged**